### PR TITLE
Make database table

### PR DESCRIPTION
**Pull request by riemannulus**
Solve relationship problem

### CODE REVIEW

File: tmp/models.py [hunk 1]
```diff
@@ -1,20 +1,43 @@
...
+
+
+class Place(Base):
+    __tablename__ = 'places'
 
-    def __repr__(self):
-        return (f"<User ip={self.ip} choice={self.choice}"
-                f" timestamp={self.timestamp}>")
+    code = Column(String(40), nullable=False, primary_key=True)
+    place = Column(String(20), nullable=False)
```
Comment (Kjwon15) [inline tmp/models.py]: I think `name` is better than `place` because it is Places' name.
place.name = '~~'

**Comment by riemannulus** [comment]
- Change name 'Place.place' to 'Place.name'.
- Change name 'DATABASE_URI' to 'DATABASE_URL'.

File: tmp/__init__.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 app.register_blueprint(sky, url_prefix='/sky')
 
 
-@app.before_first_request
```
Comment (Kjwon15) [inline tmp/__init__.py]: Why remove this?

File: tmp/__init__.py [hunk 3]
```diff
@@ -19,7 +19,7 @@
...
+
 def initialize():
     engine = create_engine(DATABASE_URI)
```
Comment (Kjwon15) [inline tmp/__init__.py]: This still use `DATABASE_URI`

**Pull request merged**